outputs/loki.go

```diff
@@ -47,6 +50,7 @@ func newLokiPayload(falcopayload types.FalcoPayload, config *types.Configuration
 // LokiPost posts event to Loki
 func (c *Client) LokiPost(falcopayload types.FalcoPayload) {
 	c.Stats.Loki.Add(Total, 1)
+	c.ContentType = LokiContentType
```

Comment:
Compare to the following hunk in the current main branch:
https://github.com/falcosecurity/falcosidekick/blob/970a90504eb3d761c8cc9617a9153a9397ba5dbf/outputs/client.go#L165-L168